embedding_llm_deployment/deploy_embedding_server.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+import argparse
+from vllm import LLM, SamplingParams
```

Comment:
You don't need to use vllm sdk api to start an online server. After installing vllm, you can just run `vllm serve <model name> --task embed <other arguments>.